openshift/odo
# Fixes list output in odo directory when component is not pushed

fixes #2028 

How to test:
1. Do `odo create <type>` and don't push on the cluster
2. Do `odo list`

The output should list the not pushed component
